toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -524,3 +533,27 @@ func customizeVerityImageHelper(buildDir string, baseConfigPath string, config *
 
 	return nil
 }
+
+func isDmVerityEnabled(buildDir string, rawImageFile string) error {
+	imageConnection := NewImageConnection()
+	err := imageConnection.ConnectLoopback(rawImageFile)
+	if err != nil {
+		return err
+	}
+	defer imageConnection.Close()
```

Comment:
You should call `imageConnection.CleanClose` at the end of the function to ensure that the image file is no longer being used before continuing. (`imageConnection.Close` closes everything asynchronously.)

Reply:
Fixed, plz check.